src/index.js

```diff
@@ -1,13 +1,13 @@
 var isArray = Array.isArray
 
 export default function cc(obj) {
-  var out = ""
-
-  if (typeof obj === "string" || typeof obj === "number") return obj || ""
+  if (typeof obj === "string") return obj
+  else if (typeof obj === "number") return obj.toString()
 
+  var out = ""
   if (isArray(obj))
-    for (var k = 0, tmp; k < obj.length; k++) {
-      if ((tmp = cc(obj[k])) !== "") {
+    for (var i = 0, tmp; i < obj.length; i++) {
```

Comment:
What's the motivation here? Readability?

Reply:
Let's use `let` here as well. 🙏

Reply:
The minifier already does that to help the Javascript engine, so the motivation here is readability only.
I'd like to share with you an interesting video about how V8 works: https://www.youtube.com/watch?v=EqSRpkMRyY4

Reply:
The minifier doesn't touch the scope of the variables. But I'm ready to start using `let`.